conans/client/toolchain/cmake.py

```diff
@@ -0,0 +1,185 @@
+# coding=utf-8
+
+import os
+import textwrap
+from jinja2 import Template
+
+from conans.client.build.cmake_flags import get_generator, get_generator_platform, CMakeDefinitionsBuilder, get_toolset
+
+
+# https://stackoverflow.com/questions/30503631/cmake-in-which-order-are-files-parsed-cache-toolchain-etc
+# https://cmake.org/cmake/help/v3.6/manual/cmake-toolchains.7.html
+# https://github.com/microsoft/vcpkg/tree/master/scripts/buildsystems
+
+
+class CMakeToolchain:
+    filename = "conan_toolchain.cmake"
+    conan_adjustements = os.path.join(os.path.dirname(__file__), "conan_adjustments.cmake")
```

Comment:
Also probably a better idea to merge the `conan_adjustements.cmake` file in the main toolchain file. And keep the `conan_project_include.cmake` because it is needed to run after the project.